tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh

```diff
@@ -396,13 +396,13 @@ install_initrd() {
 	fi
 
 	export AGENT_TARBALL=$(get_agent_tarball_path)
+	export AGENT_POLICY=yes
 
 	"${rootfs_builder}" --osname="${os_name}" --osversion="${os_version}" --imagetype=initrd --prefix="${prefix}" --destdir="${destdir}" --image_initrd_suffix="${variant}"
 }
 
 #Install guest initrd for confidential guests
 install_initrd_confidential() {
-	export AGENT_POLICY=yes
```

Comment:
Should we remove this from `install_initrd_mariner` too?

Reply:
For the sake of consistency, yes.